dashboard/utils.py

```diff
@@ -277,6 +282,47 @@ def count_courses_passed(self):
                 passed_courses.add(self.edx_key_course_map[course_id])
         return len(passed_courses)
 
+    def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
+        """
+        Get the pearson exam status for the user / program combo
+        """
+        if self.program.exam_series_code is None:
+            return ""
+        else:
```

Comment:
Code style nit: I'd probably drop the `else` so you don't have to indent the majority of the function impl.